<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PapelWidget</class>
 <widget class="QWidget" name="PapelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="papel_grid">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="sysid_lbl">
     <property name="text">
      <string>SysID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sysid_sb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="senha_lbl">
     <property name="text">
      <string>Senha:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="senha_edt"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="validade_lbl">
     <property name="text">
      <string>Validade:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="validade_dte">
     <property name="displayFormat">
      <string>yyyy-MMM-dd hh:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="limconexao_lbl">
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Conexões:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="limconexao_sb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Atributos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="superusr_chk">
        <property name="text">
         <string>Superusuário</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="herdarperm_chk">
        <property name="text">
         <string>Herdar Permissões</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="criarbd_chk">
        <property name="text">
         <string>Criar Banco de Dados</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="permitirlogin_chk">
        <property name="text">
         <string>Permitir Login</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="criarusr_chk">
        <property name="text">
         <string>Criar Usuários/Grupos</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="senhacripto_chk">
        <property name="text">
         <string>Senha Criptografada</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTabWidget" name="membros_twg">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="membros_tab">
      <attribute name="title">
       <string>Membros</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="membro_tab">
      <attribute name="title">
       <string>Membro de</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="membros_adm_tab">
      <attribute name="title">
       <string>Membros (Administradores)</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
